src/main/java/mops/persistence/permission/DirectoryPermissions.java

```diff
@@ -48,8 +66,25 @@ public DirectoryPermissions(@NonNull Set<DirectoryPermissionEntry> permissions)
      * @param id          database id
      * @param permissions role permissions
      */
-    public DirectoryPermissions(Long id, @NonNull Set<DirectoryPermissionEntry> permissions) {
-        this.id = id;
-        this.permissions = permissions;
+    DirectoryPermissions(Long id, Set<DirectoryPermissionEntry> permissions) {
+        this(id, false, permissions, null, null);
+    }
+
+    /**
+     * Get the creation time.
+     *
+     * @return creation time
+     */
+    public Instant getCreationTime() {
+        return creationTime == null ? Instant.EPOCH : creationTime.toInstant();
+    }
+
+    /**
+     * Get the last modified time.
+     *
+     * @return last modified time
+     */
+    public Instant getLastModifiedTime() {
+        return lastModifiedTime == null ? Instant.EPOCH : lastModifiedTime.toInstant();
     }
```

Comment:
Do you test this?

Reply:
just a simple getter, gets implicitly tested by the equals